pages/pools/index.tsx

```diff
@@ -322,9 +335,9 @@ const Pools: FC = () => {
 				/>
 				<Table
 					pageIndex={pageIndex}
-					noResultsMessage={poolsQuery.isSuccess && (data?.length ?? 0) === 0 ? 'no results' : null}
+					noResultsMessage={poolsQueryWithNetwork?.length === 0 ? 'no results' : null}
 					data={data && data.length > 0 ? data : []}
-					isLoading={poolsQuery.isLoading}
+					isLoading={!poolsQueryWithNetwork?.length}
```

Comment:
hmm... so we will show results as each network returns data. i guess that makes more sense than making sure all the queries have loaded. The UI might update with new data coming in but that seems ok.

Reply:
nah... you're right. Will create a helper to check when all queries status !== loading